geoh5py/ui_json/constants.py

```diff
@@ -37,42 +38,42 @@
     "workspace": None,
 }
 ui_validations = {
-    "association": {"types": [str, [str]], "values": ["Vertex", "Cell"]},
+    "association": {"types": [str, type(None)], "values": ["Vertex", "Cell"]},
     "dataGroupType": {
-        "types": [str],
+        "types": [str, type(None)],
         "values": [
             "Multi-element",
             "3D vector",
             "Dip direction & dip",
             "Strike & dip",
         ],
     },
-    "dataType": {"types": [str], "values": ["Float", "Text", "Integer"]},
-    "dependency": {"types": [str]},
-    "dependencyType": {"types": [str], "values": ["enabled", "disabled"]},
-    "enabled": {"types": [bool]},
-    "group": {"types": [str]},
-    "label": {"types": [str], "required": True},
-    "main": {"types": [bool]},
-    "meshType": {"types": [str, UUID]},
-    "optional": {"types": [bool]},
-    "parent": {"types": [str, UUID]},
+    "dataType": {"types": [str, type(None)], "values": ["Float", "Text", "Integer"]},
```

Comment:
That's not allowed for ui.json, those fields have to be one of the `values`

Reply:
you're right, there's no need for a types validation if there's also a values validation.  Removed the former.